packages/openneuro-app/src/scripts/dataset/components/dataset-event-list.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react"
+import { DatasetEventItem } from "./dataset-event-item"
+
+interface ProcessedDatasetEvent {
+  id: string
+  note: string
+  timestamp: string
+  user: {
+    id: string
+    name: string
+    email: string
+    orcid: string
+  }
+  event: {
+    type: string
+    requestId?: string
+    status?: string
+  }
+  success: boolean
+  hasBeenRespondedTo?: boolean
+  responseStatus?: string
+}
```

Comment:
The event type seems to be defined a few times. It would be good to unify these and import the shared type.